src/views/GlossaryPage.vue

```diff
@@ -0,0 +1,207 @@
+
+    <DashboardCard>
+      <template #title>Data Variables</template>
+      <template #content>
+        <div class="px-4">
+          <GlossaryDefinitions
+            v-for="data in dataAcronyms"
+            :key="data.variable"
+            :term="data.variable"
+            :unit="data.units"
+          >
+            {{ data.description }}
+          </GlossaryDefinitions>
+        </div>
+      </template>
+    </DashboardCard>
+
+    <DashboardCard>
+      <template #title>Other Acronyms</template>
+      <template #content>
+        <div class="px-4">
+          <GlossaryDefinitions
+            v-for="model in modelAcronyms"
+            :key="model.acronym"
+            :term="model.acronym"
+          >
+            {{ model.name }}
+          </GlossaryDefinitions>
+        </div>
+      </template>
+    </DashboardCard>
+  </DashboardLayout>
+</template>
+
+<script setup lang="ts">
+import DashboardLayout from "@/layouts/dashboard.vue";
+import DashboardCard from "@/components/base/DashboardCard.vue";
+import GlossaryDefinitions from "@/components/base/GlossaryDefinitions.vue";
+import { sortByProperty } from "../utils/utils";
+
+const agencyAcronyms = [
+  { acronym: "RIDDC", name: "Rhode Island Data Discovery Center" },
+  {
+    acronym: "RI C-AIM",
+    name: "Rhode Island Consortium for Coastal Ecology Assessment, Innovation, and Modeling",
+  },
+  {
+    acronym: "RIDEM-OWR",
+    name: "Rhode Island Department of Environmental Monitoring-Office of Water Resources",
+  },
+  { acronym: "NBFSMN", name: "Narragansett Bay Fixed Site Monitoring Network" },
+  {
+    acronym: "URI-GSO",
+    name: "University of Rhode Island, Graduate School of Oceanography",
+  },
+  { acronym: "NBC", name: "Narragansett Bay Commission" },
+  { acronym: "RWU", name: "Roger Williams University" },
+  { acronym: "NBEP", name: "Narragansett Bay Estuary Program" },
+  { acronym: "URI-CI", name: "University of Rhode Island-Coastal Institute" },
+
+  {
+    acronym: "MassDEP",
+    name: "Massachusetts Department of Environmental Protection",
+  },
+  { acronym: "SNEP", name: "Southern New England Estuary Program" },
+  {
+    acronym: "NERACOOS",
+    name: "Northeast Regional Association Coastal Ocean Observing Systems",
+  },
+];
+
+const modelAcronyms = [
+  {
+    acronym: "ERDDAP",
+    name: "Environmental Research Division Data Access Program",
+  },
+  { acronym: "OSOM", name: "Ocean State Ocean Model" },
+  { acronym: "ROMS", name: "Regional Ocean Modelling System" },
+  { acronym: "NABATS", name: "Narragansett Bay Time Series" },
+];
+
+const dataAcronyms = [
+  {
+    variable: "Chlorophyll",
+    units: "µg L-1",
+    description:
+      "Concentration of chlorophyll in sea water, Relative Fluorescence Units",
+  },
+  {
+    variable: "depth",
+    units: "m",
+    description: "Topographic depth at buoy location",
+  },
+  {
+    variable: "DepthSurface",
+    units: "m",
+    description: "Measured water depth under buoy",
+  },
```

Comment:
are these meaningfully different? or could they be combined into `Depth`?

Reply:
These do seem to differ, it would be great to get some clarification on how each is measured. You helped catch an error though, I should have deleted the `Surface` and it should just be `depth` and `Depth`

![image](https://user-images.githubusercontent.com/24841909/161617200-dd27721b-4f30-4e2d-a3aa-6dc5c141b130.png)

Reply:
Is there a `DepthSurface` for that dataset? I wonder if `depth == DepthSurface`